### ISSUE TITLE
Wikipedia Page: TL;DR: Please add at least one original sentence each to the wikipedia article!

### ISSUE THREAD

**Issue by mpdehnel** [tamarin-prover/manual#38]
**_TL;DR: Please add at least one original sentence each to the [wikipedia article](https://en.wikipedia.org/wiki/Tamarin_Prover)!_**

So this isn't strictly an issue for the manual, but I thought I'd flag it here to make people aware.

Sadly, some heartless wikipedia editor has gone through and entirely gutted the [Wikipedia article on Tamarin](https://en.wikipedia.org/wiki/Tamarin_Prover), citing the (sadly correct) fact that the `CC BY-NC-SA` licence that the manual is released under isn't strictly compatible with Wikipedia's requirement (they don't allow the No Commercial bit). 
To add insult to injury, having gutted the article, they've then slapped a great big 'may not meet notability requirements' tag on it, which is patently false, but they can justify it because the person removed all the content demonstrating notability… (angry reacts only).

**PLEASE** oh wonderful people of Tamarin-land, could I beg you to all spend just FIVE minutes today or tomorrow, writing (say) one sentence each on the page? If you can cite a source for it, even better. Or, perhaps just link to a major paper that used Tamarin (to show utility / notability)? I don't care what; it doesn't matter if the content is later merged or edited, but just any useful (non-copyrighted, originally written material especially for this page) would be amazing.

Thank you!

Martin

**Comment by rsasse** [comment]
I think this issue is not current anymore, so I am closing it. Speak up if you disagree.
